pkg/authorization/authorizer/messages_test.go

```diff
@@ -10,16 +10,16 @@ import (
 func TestDefaultForbiddenMessages(t *testing.T) {
```

Comment:
Add tests that have all valid fields from `AttributesRecord` filled out per `IsResourceRequest` and `!IsResourceRequest`.

 This will test the new groups path and will fail in the future if we start using the other fields in the messages.